src/cockpit/389-console/src/lib/ldap_editor/lib/utils.jsx

```diff
@@ -71,7 +71,7 @@ export function getUserSuffixes (serverId, suffixCallback) {
   ];
   log_cmd("getUserSuffixes", "list suffixes", suffixCmd);
   cockpit
-    .spawn(suffixCmd, { superuser: true })
+    .spawn(suffixCmd, { superuser: true, err: "message" })
```

Comment:
I'm not sure if it's related to this PR, but I get a browser crash if I provoke a search error in LDAP Browser like that:
[Screencast from 02-09-2023 09:39:54 AM.webm](https://user-images.githubusercontent.com/5634892/217894728-1051e541-e410-490f-bbb4-11476abba89a.webm)


The error:

```
CMD: updateEntryRows: Checking if entry is activated ==> dsidm -j ldapi://%2fvar%2frun%2fslapd-spi.socket -b dc=example,dc=com account entry-status dc=example,dc=com
index.js:2 getBaseLevelEntryAttributes failed with exit_status = 32
index.js:2 getBaseLevelEntryAttributes error message = SASL/EXTERNAL authentication started
SASL username: gidNumber=0+uidNumber=0,cn=peercred,cn=external,cn=auth
SASL SSF: 0
No such object (32)
index.js:2 CMD: getOneLevelEntries:  ==> ldapsearch -LLL -o ldif-wrap=no -Y EXTERNAL -b dc=example,dc=com -H ldapi://%2fvar%2frun%2fslapd-spi.socket -s one (|(objectClass=*)(objectClass=ldapSubEntry)) -l 10 -z 2000 1.1 numSubordinates modifyTimestamp objectclass
index.js:2 FAIL err.exit_status ==> 32
index.js:2 FAIL err.message ==> SASL/EXTERNAL authentication started
SASL username: gidNumber=0+uidNumber=0,cn=peercred,cn=external,cn=auth
SASL SSF: 0
No such object (32)
index.js:2 Uncaught TypeError: Cannot read properties of null (reading 'map')
    at index.js:2:2781573
    at Function.<anonymous> (index.js:2:2385363)
    at cockpit.js:1:12867
    at cockpit.js:1:12978
    at L (cockpit.js:1:12383)
```

Reply:
@droideck I don't think it's related because this PR only affects dsconf/dsctl/dsidm.  The LDAP browser directly calls ldapsearch.  So it looks like a new bug to me....

Reply:
> @droideck I don't think it's related because this PR only affects dsconf/dsctl/dsidm. The LDAP browser directly calls ldapsearch. So it looks like a new bug to me....

Sure, this PR looks good then!